tests/actions/SessionTest.ts

```diff
@@ -105,4 +106,41 @@ describe('Session', () => {
         TestHelper.signInWithTestUser()
             .then(TestHelper.signOutTestUser)
             .then(() => expect(PushNotification.deregister).toBeCalled()));
+
+    test('ReconnectApp should push request to the queue', () => {
+        return TestHelper.signInWithTestUser()
+            .then(() => Onyx.set(ONYXKEYS.NETWORK, {isOffline: true}))
+            .then(() => {
+                App.confirmReadyToOpenApp();
+                App.reconnectApp();
+            })
+            .then(waitForBatchedUpdates)
+            .then(() => {
+                expect(PersistedRequests.getAll().length).toBe(1);
```

Comment:
Let's also add a check to make sure that the persisted request is indeed `ReconnectApp`:

```
expect(PersistedRequests.getAll()[0].command).toBe(WRITE_COMMANDS.RECONNECT_APP);
```